test/unittest/unittest_nntrainer_models.cpp

```diff
@@ -327,6 +327,7 @@ GraphWatcher::GraphWatcher(const std::string &config, const bool opt) :
   }
 
   loss_node = NodeWatcher(graph.back());
+  expected_loss = 0.0;
```

Comment:
please set this in the initialization list - line 295.

Reply:
Done